components/checkbox/stories/checkbox.mdx

```diff
@@ -0,0 +1,28 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
```

Comment:
I'm not sure this requires a custom mdx file because the template can handle a lot of this.

On line 6 of the checkbox.stories.js file, you can update the description to this:

```
/**
 * Checkboxes allow users to select multiple items from a list of individual items, or mark one individual item as selected.
 * 
 * ### Usage notes
 * 
 * Checkboxes should not be used on their own. They should always be used within a [Field group](/docs/components-field-group--docs). Invalid checkboxes are indicated with an alert [Help text](/docs/components-help-text--docs) when included in a Field group.
 * 
 * When the label is too long for the horizontal space available, it wraps to form another line.
 */
```

Now if you move or update this mdx file not to pick up, you can see the auto template will render much of this information.

Reply:
If you look at the last few commits (https://github.com/adobe/spectrum-css/pull/2793/commits/b15b4d34beba86563643da1d8e1b09e69a8ce0d7, https://github.com/adobe/spectrum-css/pull/2793/commits/b5cb3f9451b756e63c0eac1d78431f215a154e27), I haven't quite removed the MDX files for coach indicator or checkbox yet, but I have seen your suggestions!

I tested out deleting the coach indicator MDX, and I have some thoughts I wanted to bring up. The docs template auto-formats the page differently than the layout/format that we've used for all the other docs migrations that have already merged- I think there's maybe 15ish. I have a couple of screenshots to show the layout difference with and without the MDX file:

<img width="600" alt="Screenshot 2024-06-04 at 5 56 13 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/4c34042b-d76d-427b-abad-ada605e287ac">
<img width="600" alt="Screenshot 2024-06-04 at 5 55 54 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/c7b88519-4747-47b5-ab38-4bc1bc33a997">

To match the layout of the merged docs (like Accordion, Avatar, Combobox, Calendar, etc.), would you rather me update `DocumentationTemplate.mdx`? As a bonus, the layout of those merged components closely reflect the docs site layout, too, so maybe it's worth updating `DocumentationTemplate.mdx`.

Some ideas we had around editing the docs template were:
-  swapping `<Controls />` for `<ArgTypes />`
-  remove `<Primary />` so it's just included in `<Stories />` instead
- move `<Stories />` towards the top

Here's what we can achieve if we edit the docs template (I removed the coach indicator MDX file in this case):
<img width="600" alt="Screenshot 2024-06-05 at 12 17 30 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/14b4d16c-9fef-4ad4-80d4-4c2bea62fd88">

If I didn't explain this well, let me know and we can pair!